README.md

```diff
@@ -17,14 +27,75 @@ For the hardware and software prerequiste, please refer to [PyTorch Prerequisite
 * Intel® Deep Learning Essentials: Install a subset of Intel® oneAPI components needed for building and running PyTorch.
 
-## Build
+## Build and install
+
+This project cannot be built or installed as a stand-alone. This project gets built when PyTorch is built with XPU backend support.
 
-Need to built this project as a submodule of PyTorch, after install Intel GPU Driver and Intel Deep Learning Essentials.
+**To install PyTorch with XPU backend from pre-built binary packages**, use one of the available distribution channels:
+
+* For release builds:
+
+```
+pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/xpu
+```
+
+* For nightly builds
+
+```
+pip3 install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu
+```
+
+**To build PyTorch with XPU backend from sources**, refer to the [Intel GPU Support](https://github.com/pytorch/pytorch/blob/main/README.md#intel-gpu-support) section of the [PyTorch documentation](https://github.com/pytorch/pytorch/blob/main/README.md#from-source). In summary, the PyTorch build for XPU backend can be triggered as follows:
 
 ```bash
-# Setup PyTorch source project. torch-xpu-ops is included by default.
-python setup.py install
+git clone https://github.com/pytorch/pytorch.git && cd pytorch
+pip install -r requirements.txt
+python setup.py install >log.txt 2>&1
+```
+
+Look for the following lines in the log file indicating that XPU backend is being built:
+
 ```
+$ cat log.txt | grep -E "(USE_XPU|USE_XCCL)\s*:"
+--   USE_XPU               : 1
+--   USE_XCCL              : ON
+```
+
+If building from sources, note the following environment variables which control the PyTorch XPU backend build:
+
+| Environment variable | Default | Notes |
+| --- | --- | --- |
+| `USE_XPU` | `ON` | Enables XPU backend support |
+| `USE_XCCL` | `ON` (>= PT2.8) | Enables XCCL distributed backend support |
+| `TORCH_XPU_ARCH_LIST` | depends on the PT and OS versions | Build SYCL kernels for specified platform(s) |
+
+The `TORCH_XPU_ARCH_LIST` allows to specify a comma separated list of platforms for which SYCL kernels will be built. This helps to reduce the build time. This option does not affect oneDNN- or oneMKL-based operators. Note that if PyTorch is executed on a platform that the SYCL kernels were not built for, then they are compiled just-in-time (JIT).
+
+## Verification
+
+Once PyTorch is built or installed, verify that PyTorch XPU backend is available as follows:
+
+```
+$ python3 -c "import torch; print(torch.xpu.is_available())"
+True
+python3 -c "import torch; print(torch.distributed.distributed_c10d.is_xccl_available())"
+True
+```
+
+# FAQ
+
+Some confusion might arise about relationship of this repostory and PyTorch XPU backend in general with [Intel® Extension for PyTorch (IPEX)](https://github.com/intel/intel-extension-for-pytorch). See answers on that below.
```

Comment:
IPEX is not an valid name, please use full name, not IPEX. And Intel Extension for PyTorch is deprecating, even not highlight the question is fine here.